src/v/cluster/cluster_utils.h

```diff
@@ -38,32 +39,37 @@ patch<broker_ptr> calculate_changed_brokers(
   const std::vector<broker_ptr>& new_list,
   const std::vector<broker_ptr>& old_list);
 
+class getTopicName {
+public:
+    getTopicName(){};
+    virtual double operator()(double left, double right) = 0;
+};
```

Comment:
`getTopicName` isn't used in this commit. It should either be removed, or added in a different commit, as it does seem to also be unrelated to the commit's purpose which is to `Modify create_topic_results to reduce duplicated code."

Reply:
It seems unused in all commits, I think it can be removed without issue

Reply:
It's unrelated to my code changes, I changed these lines just to pass the clang lint test.

Reply:
Not sure I follow, the clang linter test is just to ensure that everything is formatted correctly. You can use the `clang-format` binary in `./vbuild/llvm/src/clang/tools/clang-format` to format your code before its pushed to github